clients/drcachesim/tools/external/example/CMakeLists.txt

```diff
@@ -115,3 +82,36 @@ set(EMPTY_CONFIG_BUILD ${PROJECT_BINARY_DIR}/tools/empty.drcachesim)
 write_config_file(empty ${EMPTY_CONFIG_BUILD} clients/${INSTALL_LIB})
 write_config_file(empty ${EMPTY_CONFIG_INSTALL} tools/${INSTALL_LIB})
 DR_install(FILES "${EMPTY_CONFIG_INSTALL}" DESTINATION ${INSTALL_CLIENTS_BASE})
+
+
+##################################################
+# Tests
+
+set(test_seconds 90)
+
+if (X86 AND X64 AND ZIP_FOUND)
+# Simple test to ensure empty launcher keeps working.
+  set(trace_dir
+    "${PROJECT_SOURCE_DIR}/clients/drcachesim/tests/drmemtrace.threadsig.x64.tracedir")
+  add_test(NAME tool.empty_launcher
+    COMMAND empty_launcher -trace ${trace_dir})
+  set_tests_properties(tool.empty_launcher PROPERTIES TIMEOUT ${test_seconds})
+
+  # Simple test to ensure empty tool can be loaded by drcachesim.
+  file(READ "${CMAKE_CURRENT_SOURCE_DIR}/empty.templatex" empty_regex)
```

Comment:
For these simple matches meant to work as an external project and so not using the DR test .templatex infrastructure, we could simplify further by removing the file and just putting the two line string directly here as a literal.  Ditto for the other file.